.github/workflows/update_build_status.yml

```diff
@@ -74,7 +74,8 @@ jobs:
                           check_run_id: cr.id,
                           output: cr.output,
                           status: run.data.status,
-                          conclusion: run.data.conclusion
+                          conclusion: run.data.conclusion,
+                          details_url: run.data.details_url
```

Comment:
It links "View more details on GitHub Actions" to the workflow run in forked repository:
![Screen Shot 2021-04-19 at 9 16 26 PM](https://user-images.githubusercontent.com/6477701/115237290-5da12280-a157-11eb-8d6b-91c90d0dace2.png)

This is unrelated minor change.